Common/DataModel/OccupancyTables.h

```diff
@@ -0,0 +1,240 @@
+DECLARE_SOA_COLUMN(MeanOcc_NTrackITS_TPC_A_Unfm_80, meanOcc_NTrackITS_TPC_A_Unfm_80, float);
+DECLARE_SOA_COLUMN(MeanOcc_NTrackITS_TPC_C_Unfm_80, meanOcc_NTrackITS_TPC_C_Unfm_80, float);
+
+DECLARE_SOA_COLUMN(MeanOccRobust_T0V0Prim_Unfm_80, meanOccRobust_T0V0Prim_Unfm_80, float);
+DECLARE_SOA_COLUMN(MeanOccRobust_FDDT0V0Prim_Unfm_80, meanOccRobust_FDDT0V0Prim_Unfm_80, float);
+DECLARE_SOA_COLUMN(MeanOccRobust_NtrackDet_Unfm_80, meanOccRobust_NtrackDet_Unfm_80, float);
+
+DECLARE_SOA_COLUMN(WeightMeanOcc_Prim_Unfm_80, weightMeanOcc_Prim_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_FV0A_Unfm_80, weightMeanOcc_FV0A_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_FV0C_Unfm_80, weightMeanOcc_FV0C_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_FT0A_Unfm_80, weightMeanOcc_FT0A_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_FT0C_Unfm_80, weightMeanOcc_FT0C_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_FDDA_Unfm_80, weightMeanOcc_FDDA_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_FDDC_Unfm_80, weightMeanOcc_FDDC_Unfm_80, float);
+
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrack_PVC_Unfm_80, weightMeanOcc_NTrack_PVC_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrack_ITS_Unfm_80, weightMeanOcc_NTrack_ITS_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrack_TPC_Unfm_80, weightMeanOcc_NTrack_TPC_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrack_TRD_Unfm_80, weightMeanOcc_NTrack_TRD_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrack_TOF_Unfm_80, weightMeanOcc_NTrack_TOF_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrackSize_Unfm_80, weightMeanOcc_NTrackSize_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrackTPC_A_Unfm_80, weightMeanOcc_NTrackTPC_A_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrackTPC_C_Unfm_80, weightMeanOcc_NTrackTPC_C_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrackITS_TPC_Unfm_80, weightMeanOcc_NTrackITS_TPC_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrackITS_TPC_A_Unfm_80, weightMeanOcc_NTrackITS_TPC_A_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrackITS_TPC_C_Unfm_80, weightMeanOcc_NTrackITS_TPC_C_Unfm_80, float);
+
+DECLARE_SOA_COLUMN(WeightMeanOccRobust_T0V0Prim_Unfm_80, weightMeanOccRobust_T0V0Prim_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOccRobust_FDDT0V0Prim_Unfm_80, weightMeanOccRobust_FDDT0V0Prim_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOccRobust_NtrackDet_Unfm_80, weightMeanOccRobust_NtrackDet_Unfm_80, float);
+
+} // namespace trackmeanocc
+
+DECLARE_SOA_TABLE(TrackMeanOccs, "AOD", "TRACKMEANOCCS", o2::soa::Index<>,
+                  o2::aod::trackmeanocc::TrackId,
+                  o2::aod::trackmeanocc::CollisionId,
+                  o2::aod::trackmeanocc::OccId,
+                  o2::aod::trackmeanocc::BCId,
+                  o2::aod::trackmeanocc::TfId,
+                  o2::aod::trackmeanocc::BCinTF,
+                  o2::aod::trackmeanocc::MeanOcc_Prim_Unfm_80,
```

Comment:
Here the table can be divided into several joinable parts - per detector, for example. They can be filled only on-demand (i.e. only if any tasks in the workflow are requesting them specifically) and thus reduce the memory footprint.

Reply:
Regarding this, we also compare the information from various detectors to get the robust occupancy estimator. That's why we have created them in same table